### PR TITLE
spotify clone added

### LINKED ISSUE #766: Title: Spotify Clone (using HTML,CSS,JS)
### Is your proposal related to a problem? Please describe.

i wanna add this project in beginner level website folder.

### Add any other context or screenshots about the proposal request here.

_No response_

### PR DESCRIPTION
**Pull request by Laraibnehal**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

yes
### Description

(Write your answer here.)

Fixes #766

### Checklist

- [x ] I've read the contribution guidelines.
- [x ] I've checked the issue list before deciding what to submit.
- [ x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by Laraibnehal** [comment]
@Kushal997-das

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@Laraibnehal Add picture related to your project in `readme.md` file

**Comment by Laraibnehal** [comment]
@Kushal997-das

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@Laraibnehal Add some description about your project  in readme.md file also try to add this project in basic folder .

**Comment by Laraibnehal** [comment]
On it.

**Comment by Laraibnehal** [comment]
Done @Kushal997-das

File: Web Development/Basic/Spotify clone/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+to copy this in your computer: git clone  (add link)
+make changes according to yourself.
+
+Made This small spotify with HTML,CSS,JS added my own collection of favorite song you can your's.
+It's a great project for beginners and fun to make.
+
+![Spotify - Your favourite music is here - Google Chrome 1_5_2023 11_14_28 AM](https://user-images.githubusercontent.com/95610027/214508475-e46e6d43-5d7f-4141-beab-8b361d67b897.png)
```
Comment (Kushal997-das) [inline Web Development/Basic/Spotify clone/README.md:7]: No to good description . Write in a professional way .

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
Do the require changes .

File: Web Development/Basic/Spotify clone/README.md (hunk 1 above)
Comment (Laraibnehal) [reply Web Development/Basic/Spotify clone/README.md:7]: Ok

File: Web Development/Basic/Spotify clone/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Made This small spotify playlist with HTML,CSS,JS added my own collection of favorite song you can add your's too.
+
+It's a great project for beginners to gain confidence in Javascript.
+
+![Spotify - Your favourite music is here - Google Chrome 1_5_2023 11_14_28 AM](https://user-images.githubusercontent.com/95610027/214508475-e46e6d43-5d7f-4141-beab-8b361d67b897.png)
```
Comment (Kushal997-das) [inline Web Development/Basic/Spotify clone/README.md:5]: Change the description like this description seems like casual please see other project from my repo and understand how to write a good description about your project .

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
Change the description like this description seems like casual please see other project from my repo and understand how to write a good description about your project .

**Pull request closed without merging**